src/app/pages/products-list/card/card.component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component} from '@angular/core';
+import {MatCardModule} from '@angular/material/card';
+import {MatButtonModule} from '@angular/material/button';
+import {productsMock} from '../../../shared/products/products.mock';
+
+@Component({
+    selector: 'app-card',
+    standalone: true,
+    imports: [MatButtonModule, MatCardModule],
+    templateUrl: './card.component.html',
+    styleUrl: './card.component.css',
+})
+export class CardComponent {
+    readonly product = productsMock[0];
+    onClickBuy = function (event: Event) {
+        event.stopPropagation();
+        alert('onClickBuy');
+    };
```

Comment:
Почему такую инициализацию выбрал? Почему не так:
```suggestion
    onClickBuy(event: Event) {
        event.stopPropagation();
        alert('onClickBuy');
    };
```

Reply:
Рука дрогнула)